src/app/core/components/viewchartarea/viewchartarea.component.html

```diff
@@ -0,0 +1,3 @@
+<div class="wrapper">
```

Comment:
Also, please [rename to kebab-style](https://angular.io/guide/styleguide#style-02-02), i.e. view-chart-area/view-chart-area.component.html, ts, etc.
I'll rename other files.